src/js/letters/reducers/index.js

```diff
@@ -4,7 +4,7 @@ import mapValues from 'lodash/mapValues';
 const initialState = {
   letters: [],
   destination: {},
-  lettersAvailable: false,
+  lettersAvailable: undefined,
```

Comment:
Because I'm now checking if the `available` prop for each app is `true` (if fetch is successful) or `false` (if fetch is unsuccessful), I need to default this the something else for when the fetch has not completed yet.